src/modules/previewpane/powerpreview/powerpreview.rc

```diff
@@ -58,8 +58,10 @@ BEGIN
 	IDS_PRVPANE_FILE_PREV_STTNGS_GROUP_TEXT             L"Preview Pane";
 	IDS_PREVPANE_MD_BOOL_TOGGLE_CONTROLL                L"PREVPANE_MD_BOOL_TOGGLE_CONTROLL_ID"
 	IDS_PREVPANE_MD_SETTINGS_DESCRIPTION                L"Show Markdown"
+	IDS_PREVPANE_MD_SETTINGS_DISPLAYNAME                L"Makrdown Preview Handler\0"
```

Comment:
According to my understanding, an escape character is automatically added while referring to any string from the STRINGTABLE. Is there any reason for explicitly adding it?